tests/features.py

```diff
@@ -0,0 +1,42 @@
+from flask import current_app
+
+
+def enable_feature(name, value=True):
```

Comment:
The nice thing about 'set' is that on assumes it's positive (i.e. True), but it still makes sense for False. `feature_flag` doesn't make it clear what the default is, IMO